transformer_engine/pytorch/attention/multi_head_attention.py

```diff
@@ -570,15 +577,23 @@ def set_context_parallel_group(
             self.cp_size = get_distributed_world_size(cp_group)
             self.cp_rank = get_distributed_rank(cp_group)
         elif isinstance(cp_group, list):
-            assert len(cp_group) == 2, "Current implementation only supports two-level CP groups!"
+            assert len(cp_group) == 3, (
```

Comment:
This is a breaking change in the API, no? That is not good. Also, the docstring still says that it should be list of size 2.